Modules/Twp/Entities/Student.php

```diff
@@ -37,6 +37,13 @@ public function application()
         return $this->hasOne('Modules\Twp\Entities\Application', 'student_id', 'id');
     }
 
+    public function indigeneity()
+    {
+        return $this->belongsToMany(
```

Comment:
return $this->belongsToMany('Modules\Twp\Entities\IndigeneityType', 'indigeneity_student');